src/udev/udev-event.c

```diff
@@ -34,6 +34,11 @@
 #include "udev-watch.h"
 #include "user-util.h"
 
+#if HAVE_STAP
+#include <sys/sdt.h>
```

Comment:
hmm, our coding style usually mandates that `<>` includes happen before `""` includes. I'd prefer if the same would be done here, even if thta means we'll have two `#if` pairs

Reply:
Given that this file does not use anything from `sys/sdt.h` and the generated header file already includes `sys/sdt.h` it can probably be removed completely.